static/docs/command-reference/pipeline/show.md

```diff
@@ -52,6 +51,29 @@ instead of stages.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Paging the output
+
+This command's output is automatically pushed to `less` command line tool if
+`less` is runnable (installed and can be executed in your shell). To be exact,
+the actual command is `less --chop-long-lines --clear-screen`.
+
+If `less` is not runnable (usually Microsoft Windows), the ouput is simply
+printed out.
+
+> Note Microsoft Windows users may also want to read
+> [this how-to](/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less).
+
+### Providing a custom pager
+
+You can also override the default pager via `DVC_PAGER` environemnt variable.
+
+For example
+
+```
+# one-time change in Bash
+DVC_PAGER=more dvc pipeline show --ascii my-pipeline.dvc
```

Comment:
should it be `export DVC_PAGER ...`?

Reply:
1. without `export` it sets variable and forget
2. with `export` it sets variable and keeps it in the session

See
```
xliiv@alef:~/workspace/dvc-project/dvc.org$ SOME_VAR=test echo hello
hello
xliiv@alef:~/workspace/dvc-project/dvc.org$ echo $SOME_VAR

xliiv@alef:~/workspace/dvc-project/dvc.org$ export SOME_VAR=test echo hello
xliiv@alef:~/workspace/dvc-project/dvc.org$ echo $SOME_VAR
test

```

Reply:
I think it's ok, depends what you want to achieve..

Reply:
so, what would be the most common usage scenario? how would you use it personally?

Reply:
i would make it persistent in `fish` config.
ok got it.

Should i add an example which shows how to make it persistent?
Or maybe replace the current example?
which shell should i support?

Reply:
no need to put full-featured examples, just describe the essence of the process in both cases - making it persistent via changing config files + exporting for a session.

Always prefer sh/bash examples since ppl who use fish will find a way to adjust it.

Reply:
done, can you check it?

Reply:
Looks good. I'll re-review this.